uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/LeavePeriodEntitlement.php

```diff
@@ -13,6 +13,8 @@
  */
 class CRM_HRLeaveAndAbsences_BAO_LeavePeriodEntitlement extends CRM_HRLeaveAndAbsences_DAO_LeavePeriodEntitlement {
 
+  use CRM_HRLeaveAndAbsences_ACL_LeaveApproverTrait;
+
```

Comment:
We should remove the reference to `CRM_HRLeaveAndAbsences_API_Query_ACLQueryHelper as ACLQueryHelper` at the beginning of this file since its no longer necessary.